### PR TITLE
have vhd cache multiple versions of runc

### PR DESCRIPTION
**Pull request by eriksywu**

### CODE REVIEW

File: vhdbuilder/packer/packer_source.sh [hunk 1]
```diff
@@ -103,3 +103,15 @@ cpAndMode() {
   src=$1; dest=$2; mode=$3
   DIR=$(dirname "$dest") && mkdir -p ${DIR} && cp $src $dest && chmod $mode $dest || exit $ERR_PACKER_COPY_FILE
 }
+
+downloadDebPkgToFile() {
```
Comment (xuto2) [inline vhdbuilder/packer/packer_source.sh:107]: shouldn't be here, it should be in cse_helper?

File: vhdbuilder/packer/install-dependencies.sh [hunk 2]
```diff
@@ -117,8 +117,21 @@ else
...
+if [[ $OS == $UBUNTU_OS_NAME ]]; then
+  RUNC_VERSIONS="
+  1.0.0-rc92
+  1.0.0-rc95
+  "
+  for RUNC_VERSION in $RUNC_VERSIONS; do
+    downloadDebPkgToFile "moby-runc" ${RUNC_VERSION} ${RUNC_DOWNLOADS_DIR}
+    echo "  - [cached] runc ${RUNC_VERSION}" >> ${VHD_LOGS_FILEPATH}
+  done
+fi
```
Comment (xuto2) [inline vhdbuilder/packer/install-dependencies.sh]: should this be in ensureRunc() so all the runc install/caching logic stay together?

Comment (xuto2) [reply vhdbuilder/packer/install-dependencies.sh]: also rc95 is used for docker nodes as well, did Brian confirm it's ok?

File: vhdbuilder/packer/packer_source.sh (hunk 1 above)
Comment (eriksywu) [reply vhdbuilder/packer/packer_source.sh:107]: I want to separate VHD building code (i.e here) with cse code since downloading to file is vhd-only

Comment (eriksywu) [reply vhdbuilder/packer/packer_source.sh:107]: moving to cse_helper

File: vhdbuilder/packer/install-dependencies.sh (hunk 2 above)
Comment (eriksywu) [reply vhdbuilder/packer/install-dependencies.sh]: ensureRunc is for installing a specific runc version though - this is to just cache runc binaries during vhd build. i think they should be separate. 

checking w/ brian

Comment (eriksywu) [reply vhdbuilder/packer/install-dependencies.sh]: docker w/ r95 is good and recommended

**Pull request merged**